docker/dockerfile-build-ubuntu-14.04

```diff
@@ -36,6 +36,7 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-ins
     libc++abi-dev \
     libc++-dev \
     hsa-rocr-dev && \
+    rocm-device-libs \
```

Comment:
This should come before the &&

Reply:
Good catch.  I've corrected this in the latest commit.